src/Sulu/Bundle/LocationBundle/Resources/views/Template/content-types/location.html.twig

```diff
@@ -7,8 +7,8 @@
      data-type-instance-name="{{ id|raw }}"
      data-instance-name="{{ id|raw }}"
      data-form="true"
-     data-aura-countries="{{ params.countries|json_encode }}"
-     data-aura-map-providers="{{ params.mapProviders|json_encode }}"
+     data-aura-countries="{{ parameterToKeyValue(params.countries.value)|json_encode }}"
```

Comment:
yes thats right ... shouldn´t we do it in another PR? this is not really related to smart content. i have created a new issue https://github.com/sulu-cmf/sulu/issues/835